test/functional/test_site/expected/testImportVariables._include_.html

```diff
@@ -6,7 +6,7 @@
 <p>Test import variables that itself imports other variables:</p>
 <h2 id="trying-to-access-a-page-variable">Trying to access a page variable:<a class="fa fa-anchor" href="#trying-to-access-a-page-variable" onclick="event.stopPropagation()"></a></h2>
 <p>There should be something red below:</p>
-<div style="color:red">This is a page variable from <a href="http://variablestoimport.md">variablestoimport.md</a></div>
```

Comment:
Is the removal of the anchor tags expected?

Reply:
It feels like there was a bug with our previous implementation that got `variablestoimport.md` to be rendered as a link. The new version seems to be the correct one.

Reply:
yup, this is the source `<variable name="pagevar">This is a page variable from variablestoimport.md</variable>`, it was there at first due to the removed line 125 of `parser.js` which does `md.renderInline` on it.
All md rendering is done in the `render` stage, so this isn't necessary.It gets removed because in the source file, this is the code:
```
<div style='color:red'>{{pagevar}}</div>
```

by commonmark spec `pagevar` which contains the anchor dosen't get markdown treatment

Reply:
> Just to confirm, I assume you've verified this with the new changes? 🙂

just checked again on the latest master (clean rebase below), no changes save for logs & timestamps!

Reply:
I'll get this in then :grin: